components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -229,6 +231,12 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         return option.username;
     }
 
+    private filterOutDeletedAndExcludedAndNotInTeamUsers = (users: UserProfile[], excludeUsers: UserProfile[]): UserProfileValue[] => {
+        return users.filter((user) => {
+            return user.delete_at === 0 && !excludeUsers.map((user) => user.id).includes(user.id);
+        }).map((user) => user as UserProfileValue);
```

Comment:
Instead of calling `Array.map` here to change the type of the result to `UserProfileValue[]`, I think it would be easier to just use `as UserProfileValue[]` on the whole array instead since that'd also avoid creating another array